app/models/question.rb

```diff
@@ -1,2 +1,4 @@
 class Question < ApplicationRecord
+  belongs_to :test
+  has_many :answers, dependent: :nullify
```

Comment:
```suggestion
  has_many :answers, dependent: :destroy
```
вряд ли нужны ответы на вопрос, который удалили